### PR TITLE
Card

### PR DESCRIPTION
**Pull request by som749898**
please review my card component

### CODE REVIEW

**Review by Creator-singh — APPROVED** [review]
looks good to me great work 🚀

File: CSS/utility.css [hunk 1]
```diff
@@ -107,7 +119,9 @@
   margin-right: 1.875rem;
 }
 
-
+.mg-top-1x {
+  margin-top: 0.625rem;
+}
```
Comment (Aditi0505) [inline CSS/utility.css]: Suggestion: remove extra spacing.

File: CSS/utility.css [hunk 2]
```diff
@@ -133,14 +147,19 @@
   max-height: 9.375rem;
   overflow: scroll;
   overflow-x: hidden;
+  /* overflow-x: visible;
+  max-width: 60rem; */
```
Comment (Aditi0505) [inline CSS/utility.css:148]: Suggestion: please remove commented code if not necessary.

File: CSS/utility.css [hunk 3]
```diff
@@ -133,14 +147,19 @@
...
   background-color: var(--background-cover);
   border: 1px solid var(--secondary-color);
 }
 
-
 .round {
   border-radius: 50%;
+}
+
+.width-300 {
```
Comment (Aditi0505) [inline CSS/utility.css:160]: Can have better class name. What is 300 specifying here?  Let me know your thoughts on this.

File: Components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.card {
+  position: relative;
+  border: none;
+}
+
+.card-img {
+  
+}
```
Comment (Aditi0505) [inline Components/Card/card.css]: Can remove this if there there is no styling here.

File: Components/Card/card.html [hunk 5]
```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>SKY UI</title>
```
Comment (Aditi0505) [inline Components/Card/card.html:8]: Can add more meaningful title. Eg. Sky | Card

**Review by Aditi0505 — APPROVED** [review]
Left few small suggestions. Rest looks good to me.
Also, please provide description and screenshots as well for better visualization.

File: Components/Card/card.html [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+            <li><a href="/Components/Card/card.html">Card</a></li>
+            <li><a href="/Components/Image/image.html">Image</a></li>
+            <li><a href="/Components/Input/input.html">Input</a></li>
+            <li><a href="#">List</a></li>
+            <li><a href="#">Toast</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="content mg-1x">
+        <h1 class="heading2 py-bottom-1x">Card</h1>
```
Comment (SanketDhabarde) [inline Components/Card/card.html:51]: Same here tag `h1` but class `heading2`. Also `py` used for top-bottom padding & `px` for left-right padding in most component libraries . It is confusing if you use `py-bottom`

File: Components/Card/card.html [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+        <div class="get-started py-bottom-2x">
+          <h3 class="heading4">Get Started</h3>
+          <ul>
+            <li><a href="/Get-Started/introduction.html">Introduction</a></li>
+            <li><a href="/Get-Started/color.html">Color</a></li>
+            <li><a href="/Get-Started/typography.html">Typography</a></li>
+          </ul>
+        </div>
+        <div class="components">
+          <h3 class="heading4">Components</h3>
```
Comment (SanketDhabarde) [inline Components/Card/card.html:36]: Why class name is `heading4` but tag is `<h3></h3>`

**Review by SanketDhabarde — CHANGES_REQUESTED** [review]
Commented some changes

File: CSS/utility.css (hunk 1 above)
Comment (som749898) [reply CSS/utility.css]: corrected it

File: CSS/utility.css (hunk 2 above)
Comment (som749898) [reply CSS/utility.css:148]: I have kept it for future refference

File: CSS/utility.css (hunk 3 above)
Comment (som749898) [reply CSS/utility.css:160]: 300 means width of 300px, I did not get a better name for it

File: Components/Card/card.css (hunk 4 above)
Comment (som749898) [reply Components/Card/card.css]: corrected it

File: Components/Card/card.html (hunk 6 above)
Comment (som749898) [reply Components/Card/card.html:51]: I named the heading according to design system shared by tanay

**Pull request merged**